### PR TITLE
bless on arm64 Fixes #204

### LINKED ISSUE #204: MacOS 12.3 on M1 device: bless: The 'openfolder' is not supported on Apple Silicon devices.
Got this error after MacOS update

```
Error: Command failed: bless --folder /Volumes/App --openfolder /Volumes/App
bless: The 'openfolder' is not supported on Apple Silicon devices.
```

### PR DESCRIPTION
**Pull request by cybergrind**

### CODE REVIEW

File: lib/appdmg.js [hunk 1]
```diff
@@ -397,8 +397,10 @@ module.exports = exports = function (options) {
   pipeline.addStep('Blessing image', function (next) {
     const args = [
       '--folder', global.temporaryMountPath,
```
Comment (LinusU) [inline lib/appdmg.js:399]:
```suggestion
      '--folder', global.temporaryMountPath
```

File: lib/appdmg.js [hunk 2]
```diff
@@ -397,8 +397,10 @@ module.exports = exports = function (options) {
...
-      '--openfolder', global.temporaryMountPath
     ]
+    if (os.arch() !== 'arm64') {
```
Comment (LinusU) [inline lib/appdmg.js:401]:
```suggestion

    if (os.arch() !== 'arm64') {
```

**Review by LinusU — APPROVED** [review]
Looks great!

Just to linting nits, I thought I could apply them myself but that option seems to be missing 🤔

If you can just add them I'm happy to merge and release right away 🚀 

Thank you for working on this!

**Comment by cybergrind** [comment]
@LinusU fixed. sorry, for some reason eslint didn't catch up config

**Comment by LinusU** [comment]
Thanks! Released as [🚢 0.6.3 / 2022-03-17](https://github.com/LinusU/node-appdmg/releases/tag/v0.6.3)

**Pull request merged**